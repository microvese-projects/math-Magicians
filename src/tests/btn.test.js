import React from 'react';
import renderer from 'react-test-renderer';
import Btn from '../components/btn';

test('Should render button correctly', () => {
  const tree = renderer
    .create(
      <Btn
        value="AC"
        styled="btn"
        id="all-clear"
        onclick
      >
        Facebook
      </Btn>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
