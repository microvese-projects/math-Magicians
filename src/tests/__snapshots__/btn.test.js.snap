// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render button correctly 1`] = `
<button
  className="btn"
  id="all-clear"
  onClick={[Function]}
  type="button"
>
  AC
</button>
`;
